pages/trading/other_limits.md

```diff
@@ -2,15 +2,19 @@
 
 Subaccounts may have a limited number of open orders at any one time determined by the net collateral of the subaccount as per:
 
-| Net Collateral | Short-term orders | Long-term or Conditional orders |
-| -------------- | ----------------- | ------------------------------- |
-| < $20          | 0                 | 0                               |
-| >= $20 and < $100         | 4                 | 4                               |
-| >= $100 and < $1,000       | 8                 | 8                               |
-| >= $1,000 and < $10,000      | 10                | 10                              |
-| >= $10,000 and < $100,000     | 100               | 100                             |
-| >= $100,000      | 1000               | 200                             |
+| Net Collateral | Long-term or Conditional orders |
+| -------------- | ------------------------------- |
+| < $20          | 0                               |
+| >= $20 and < $100         | 4                               |
+| >= $100 and < $1,000       | 8                               |
+| >= $1,000 and < $10,000      | 10                              |
+| >= $10,000 and < $100,000     | 100                             |
+| >= $100,000      | 200                             |
```

Comment:
i believe these values have changed, is it possible to instead link somewhere for the latest values? that way we don't have to update this every time